pkg/packagekit/updates.jsx

```diff
@@ -245,6 +266,39 @@ function UpdatesList(props) {
     );
 }
 
+function UpdateHistory(props) {
+    if (!props.history)
+        return null;
+
+    var now = Date.now();
+
+    function formatHeading(time) {
+        if (time)
+            return cockpit.format(_("The following packages were updated $0 ago:"), moment.duration(now - time).humanize());
+        return _("The following packages were recently updated:");
+    }
+
+    function formatPkgs(pkgs) {
+        var names = Object.keys(pkgs);
+        names.sort();
+        names.shift(); // remove _time
```

Comment:
is `_time` always in here? why not delete it before sorting, then you don't require a specific sort method... or actually you could remove it by value.